README.md

```diff
@@ -1,9 +1,9 @@
 # Operators for Qiskit
 
-###  - [Qiskit developer's exp operator](https://github.com/husky-parul/glowing-quantum/tree/qiskit-dev-op/operators-examples/qiskit-dev-operator)
+###   [Qiskit developer's experience operator](https://github.com/husky-parul/glowing-quantum/tree/qiskit-dev-op/operators-examples/qiskit-dev-operator)
```

Comment:
Often READMEs describe the repo structure with links to the main sections, such as manifests, examples, doc, etc. Maybe this repo will remain small enough that this navigation help is not needed?